Project.toml

```diff
@@ -6,6 +6,7 @@ version = "0.8.0"
 [deps]
 Animations = "27a7e980-b3e6-11e9-2bcd-0b925532e340"
 Cairo = "159f3aea-2a34-519c-b102-8c37f9878175"
+Cassette = "7057c7e9-c182-5462-911a-8362d720325c"
```

Comment:
Needs also a compat entry for later ☺️

Reply:
is this manually added ? or is the something i have to do from Pkg?